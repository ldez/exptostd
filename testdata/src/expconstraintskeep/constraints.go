package expconstraintskeep

import (
	"golang.org/x/exp/constraints"
)

type _ interface {
	constraints.Ordered | constraints.Float // want "golang.org/x/exp/constraints.Ordered can be replaced by cmp.Ordered"
}
