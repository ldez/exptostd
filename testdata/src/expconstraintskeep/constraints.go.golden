package expconstraintskeep

import (
	"golang.org/x/exp/constraints"
)

type _ interface {
	cmp.Ordered | constraints.Float // want "golang.org/x/exp/constraints.Ordered can be replaced by cmp.Ordered"
}
